examples/iterators/search.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
Just a thought, may be you can combine all the iterators in 1 file. It feels fairly repetitive.

Reply:
It is...  But the point of the examples are to provide specific, easy to read use cases.  Combining all endpoints in a single file makes it harder to pick out the point of each example.  I'm happy with the way it is, but of course someone is always free to make further changes.